extensions/mssql/src/objectManagement/localizedConstants.ts

```diff
@@ -323,6 +324,45 @@ export const DatabaseScopedOptionsColumnHeader = localize('objectManagement.data
+export const AutogrowthMaxsizeText = localize('objectManagement.databaseProperties.autogrowthMaxsizeText', "Autogrowth / Maxsize");
+export const PathText = localize('objectManagement.databaseProperties.pathText', "Path");
+export const FileNameText = localize('objectManagement.databaseProperties.fileNameText', "File Name");
+export const DatabaseFilesText = localize('objectManagement.databaseProperties.databaseFilesText', "Database files");
+export const AddDatabaseFilesText = localize('objectManagement.databaseProperties.addDatabaseFilesText', "Add Database file");
+export const EditDatabaseFilesText = (fileName: string) => localize('objectManagement.databaseProperties.editDatabaseFilesText', "Edit Database file - {0}", fileName);
+export const AddButton = localize('objectManagement.databaseProperties.addButton', "Add");
+export const EditButton = localize('objectManagement.databaseProperties.editButton', "Edit");
+export const RemoveButton = localize('objectManagement.databaseProperties.removeButton', "Remove");
+export const SizeInMbText = localize('objectManagement.databaseProperties.size', "Size (MB)");
+export const EnableAutogrowthText = localize('objectManagement.databaseProperties.enableAutogrowthText', "Enable Autogrowth");
+export const FileGrowthText = localize('objectManagement.databaseProperties.fileGrowthText', "File Growth");
+export const MaximumFileSizeText = localize('objectManagement.databaseProperties.maximumFileSizeText', "Maximum File Size");
+export const InPercentAutogrowthText = localize('objectManagement.databaseProperties.inPercentAutogrowthText', "In Percent");
+export const InMegabytesAutogrowthText = localize('objectManagement.databaseProperties.inMegabytesAutogrowthText', "In Megabytes");
+export const LimitedToMBFileSizeText = localize('objectManagement.databaseProperties.limitedToMBFileSizeText', "Limited to (MB)");
+export const UnlimitedFileSizeText = localize('objectManagement.databaseProperties.unlimitedFileSizeText', "Unlimited");
+export const PercentText = localize('objectManagement.databaseProperties.percentText', "Percent");
+export const MBText = localize('objectManagement.databaseProperties.mb', "MB");
+export const NoneText = localize('objectManagement.databaseProperties.noneText', "None");
+export function AutoGrowthValueStringGenerator(isFileGrowthSupported: boolean, fileGrowth: string, isFleGrowthInPercent: boolean, maxFileSize: number): string {
+	const maxSizelimitation = maxFileSize === -1
+		? localize('objectManagement.databaseProperties.autoGrowthValueConversion.unlimited', "Unlimited")
+		: localize('objectManagement.databaseProperties.autoGrowthValueConversion.limitation', "Limited to {0} MB", maxFileSize);
+	return isFileGrowthSupported ? localize('objectManagement.databaseProperties.autoGrowthValueConversion', "By {0} {1}, {2}", fileGrowth, isFleGrowthInPercent ? PercentText : MBText, maxSizelimitation)
```

Comment:
Whenever you have a parameter to the loc string please also include a comment saying what it is for the localizers

Reply:
correct me if im wrong, for example, in current AutoGrowthValueStringGenerator loc string, parameters isFileGrowthSupported etc, do you want me to add a comment that describes this parameter?

// isFileGrowthSupported - If the file is supported the filegrowth
// fileGrowth - file growth in MB

etc. ?